app/models/employment/BenefitsViewModel.scala

```diff
@@ -21,8 +21,9 @@ import play.api.libs.json.{OFormat, __}
 import utils.EncryptedValue
 
 case class BenefitsViewModel(
+                        accommodationRelocationModel: Option[AccommodationRelocationModel] = None,
                         carVanFuelModel: Option[CarVanFuelModel] = None,
-                        accommodation: Option[BigDecimal] = None,
+                        //accommodation: Option[BigDecimal] = None,
```

Comment:
the accommodation and relocation line items that were commented out can be removed